src/cloud/benchflow/sender/sender.go

```diff
@@ -192,25 +229,37 @@ func consumeFromTopic(t TransformerSetting) {
 }
 
 // Function that constructs the arguments for a spark-submit comand for an analyser script
-func constructAnalyserSubmitArguments(scriptName string, script string, trialID string, experimentID string, SUTName string, containerID string) []string {
+func constructAnalyserSubmitArguments(scriptName string, script string, trialID string, experimentID string, SUTName string, containerID string, hostID string) []string {
 	var args []string
```

Comment:
The code shared with `constructTransformerSubmitArguments` should be in a shared method